specification/attestation/Attestation/tspconfig.yaml

```diff
@@ -0,0 +1,43 @@
+parameters:
+  "service-dir":
+    default: "sdk/attestation"
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-rulesets/data-plane"
+
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    examples-dir: "{project-root}/examples"
+    output-file: "{azure-resource-provider-folder}/Microsoft.Attestation/{version-status}/{version}/attestation.json"
+  "@azure-tools/typespec-python":
+    package-dir: "azure-security-attestation"
+    package-name: "azure-security-attestation"
+    package-pprint-name: "Attestation"
+    namespace: "azure.security.attestation"
+    package-version: "1.1.0"
+    flavor: "azure"
+    generate-test: false
+    generate-sample: false
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.Security.Attestation"
```

Comment:
Remove `package-dir` and add `emitter-output-dir: "{output-dir}/{service-dir}/{namespace}"`